### PR TITLE
Use terragrunt

### PR DESCRIPTION
**Pull request by reshmaattar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,61 @@
+
+Mochi Infrastructure (IaC)
+=========
+
 This directory contains terraform script to create,plan and manage infrastructure automatically required for building Moch App.
```
Comment (sudhir-loveshark) [inline README.md:5]:
```suggestion
This directory contains terraform scripts to create, plan and manage infrastructure required by the Mochi app.
```

File: README.md [hunk 2]
```diff
@@ -1,32 +1,61 @@
...
 
-Features
+If you are interested to learn terraform,please check out our [Getting Started guides](https://learn.hashicorp.com/terraform#getting-started) on HashiCorp's learning platform. There are also [additional guides](https://learn.hashicorp.com/terraform#operations-and-development) to continue your learning.
```
Comment (sudhir-loveshark) [inline README.md:7]:
```suggestion
If you are interested in learning more about terraform, please check out the [Getting Started guides](https://learn.hashicorp.com/terraform#getting-started) on HashiCorp's learning platform.
```

**Review by sudhir-loveshark — APPROVED** [review]

**Comment by reshmaattar** [comment]
@sudhir-loveshark Can you have look again to the same PR.I made fixes for cognito-idp module and encrypted secrete into this pr only .

File: .gitignore [hunk 3]
```diff
@@ -15,7 +15,7 @@ crash.log
 # password, private keys, and other secrets. These should not be part of version
 # control as they are data points which are potentially sensitive and subject
 # to change depending on the environment.
-*.tfvars
+.*.tfvars
```
Comment (sudhir-loveshark) [inline .gitignore:18]: What does the extra dot do here?

**Review by sudhir-loveshark — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (reshmaattar) [reply .gitignore:18]: Sometimes if we need. to skip .local.tfvars. e.g. .local.tfvars which may contain confidentials.

**Comment by reshmaattar** [comment]
Merged

**Pull request merged**